Products/ATContentTypes/content/file.py

```diff
@@ -28,6 +27,13 @@
 import logging
 
 
+try:
```

Comment:
We can even skip the try/except and require in the setup.py Products.MimetypesRegistry >= 2.1.
Anyway I prefer the try/except option for a minor release.